rpms/s3/s3rpm.spec

```diff
@@ -122,6 +122,20 @@ S3 server provides S3 REST API interface support for Motr object storage.
 %prep
 %setup -n %{name}-%{version}-%{_s3_git_ver}
 
+%pre
+if [ -f /opt/seagate/cortx/s3/conf/s3config.yaml.sample ]; then
```

Comment:
Make 2 layer wrapper script and lets remove intelligence from rpm post install

Reply:
fixed